lua-vips-1.1-10.rockspec

```diff
@@ -34,5 +47,6 @@ build = {
        ["vips.Image"] = "src/vips/Image.lua",
        ["vips.Image_methods"] = "src/vips/Image_methods.lua",
        ["vips.Interpolate"] = "src/vips/Interpolate.lua"
-   }
+   },
+   copy_directories = { "example" }
```

Comment:
Is the a point in copying the `example` directory? As in, do you expect users to navigate into their Luarocks install directory to run the example code?

Reply:
Yeah, not sure about it. I thought it was good practice to copy directories like "docs" or "examples". Looking at some widely used modules it doesn't look so. I'll remove it then.